### ISSUE TITLE
✔️ 🐞 Bug Report: RTL problem with text path widget when browsing from Google Chrome browser

### ISSUE THREAD

**Issue by elementorovizt** [elementor/elementor#17309]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

The Text Path widget changed its direction in Hebrew in the latest update of the Google Chrome browser (version 96). I saw the RTL problem you documented [here.](https://github.com/elementor/elementor/blob/a4ba3cdece9dfd52b28acd55fc110be39a4e49da/docs/modules/shapes/assets/js/frontend/handlers/text-path.md#:~:text=Attention%20Needed%20/%20Known%20Issues%3A)
It can be deduced from this that now Google Chrome is also behaving like Firefox.
Now, it is necessary to attach the Google Chrome browser to the exception that you have disabled the Firefox browser!

## Steps to reproduce
Log in to the editor, drag and drop a text-path widget, write text in Hebrew, set a text direction from right to left in the content tab (or leave it by default), and you will get the opposite direction.
[see picture.](https://prnt.sc/253o0hu)

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by elementorovizt** [comment]
As of today, the Microsoft Edge browser (version 97) is also joining this beautiful story! It should be excluded too!

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this issue was fixed in Elementor **v3.10**! 🥳 

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂

**Comment by elementorovizt** [comment]
Wow, I thank you very much on behalf of all Hebrew speakers!
You handled the problem perfectly!!
